tests/helper.js

```diff
@@ -184,4 +185,29 @@
   };
 
 
+  exports.ReceiptValidator = function ReceiptValidator(opt) {
+    opt = utils.defaults(opt, {
+      response: {status: 'ok'},
+      onRequest: function() {},
+      verifyUrl: new RegExp(
+        'https://receiptcheck-payments-alt\\.allizom\\.org/verify/'),
```

Comment:
ok! It worked. I was missing a slash which is why the non-regex wasn't working. By "sinon bug" I of course meant user error :lollipop: